addons/binding/org.openhab.binding.miele/ESH-INF/thing/fridge.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Dishwasher -->
+    <thing-type id="fridge">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="xgw3000" />
+        </supported-bridge-type-refs>
+
+        <label>Fridge</label>
+        <description>This is a Miele@Home compatible fridge</description>
+        
+        <channels>
+            <channel id="state" typeId="state" />
+            <channel id="supercool" typeId="supercool" />
+            <channel id="current" typeId="current" />
+            <channel id="target" typeId="target" />
+            <channel id="door" typeId="door" />
+            <channel id="start" typeId="switch" />
+        </channels>        
+        
+        <config-description>
+            <parameter name="uid" type="text">
+                <label>ID</label>
+                <description>The identifier identifies one certain appliance on the ZigBee network.</description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+
+    </thing-type>
+    
+    <channel-type id="state" advanced="false">
+        <item-type>String</item-type>
+        <label>Status</label>
+        <description>Current status of the appliance</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="supercool" advanced="false">
+        <item-type>String</item-type>
+        <label>Super Cool</label>
+        <description>Start Super Cooling</description>
+        <state readOnly="true"></state>
+    </channel-type>
+
+   <channel-type id="current" advanced="false">
+        <item-type>Number</item-type>
+        <label>Current Temperature</label>
+        <description>Current temperature in the fridge</description>
+        <state readOnly="true"></state>
+    </channel-type>
+        
+   <channel-type id="target" advanced="true">
+        <item-type>Number</item-type>
+        <label>Target Temperature</label>
+        <description>Target temperature to be reached by the fridge</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+    <channel-type id="door" advanced="false">
+        <item-type>Contact</item-type>
+        <label>Door</label>
+        <description>Current state of the door of the appliance</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+     <channel-type id="start" advanced="false">
+        <item-type>Switch</item-type>
+        <label>Start</label>
```

Comment:
Ah.... We should ask Miele. The funny thing is that - while reverse engineering - I  saw that a lot of appliances seem to have commands or alike that are not very logical. This is one of them, but it is part of what the appliance is returning. I have implemented it, but I have no clue what it should do.